src/pages/Homepage/Homepage.jsx

```diff
@@ -0,0 +1,174 @@
+
+      <div
+        id="expand-section"
+        className="flex md:flex-row flex-col md:w-full md:h-screen md:pl-20 pr-6 py-6 sm:p-6 gap-2"
+      >
+        <div
+          id="bg-img-container"
+          alt="two women giggling while sitting at the table in the cafe"
+          className="bg-[url('/assets/bgImg/expand.jpg')] h-[65svh] md:h-screen md:w-full bg-no-repeat bg-cover self-right sm:bg-right "
+          data-aos="zoom-in-right"
+          data-aos-delay="300"
+        ></div>
+
+        <div
+          id="expand-text-section"
+          className="flex flex-col w-full md:justify-center md:self-center self-end"
+        >
+          <h1
+            id="expand-header"
+            className="w-fit self-end flex flex-col sm:self-center justify-start md:text-3xl text-xl align-top md:self-center md:justify-center md:items-center py-1"
+            data-aos="zoom-in"
+            data-aos-delay="500"
+            data-aos-duration="2000"
+          >
+            EXPAND
+          </h1>
+
+          <span
+            className="w-11/12 md:self-center md:text-center md:text-sm text-xs sm:text-center self-end text-right"
+            data-aos="zoom-in-left"
+            data-aos-delay="300"
+            data-aos-duration="2000"
+          >
+            expand your network for growth and opportunities
+          </span>
+        </div>
+      </div>
+
+      <div
+        id="join-section"
+        className="flex md:flex-row-reverse md:w-full md:h-full flex-col pl-6 gap-4 sm:p-6 md:pr-20"
+      >
+        <div
+          id="bg-img-container"
+          alt="lady sitting at the table with her feet on the chair with a bowl of pasta, holding her phone in the left hand"
+          className="md:w-1/2 bg-[url('/assets/bgImg/join.jpg')] md:h-screen bg-no-repeat bg-right w-full bg-cover self-end h-[60svh]"
+          data-aos="fade-left"
+          data-aos-delay="300"
+          data-aos-duration="1500"
+        ></div>
+        <div
+          id="join-text-section"
+          className="flex flex-col justify-center items-center pr-3 md:w-1/2"
+        >
+          <h1
+            id="join-header"
+            className="w-fit md:self-center  flex flex-col  md:text-3xl text-xl text-start pb-1"
+            data-aos="fade-down"
+            data-aos-delay="400"
+          >
+            JOIN
+          </h1>
+
+          <span
+            className="w-4/5 md:self-center flex flex-col items-center justify-center md:w-11/12 sm:self-center sm:w-11/12 text-xs pb-5"
+            data-aos="fade-up"
+            data-aos-delay="200"
+            // data-aos-duration="1000"
+          >
+            LUNCHbuddy and never eat lunch alone
+          </span>
+          <Link
+            to={'/register'}
+            className="border-2 self-center border-orange-400 rounded-3xl px-10 py-1 text-xs md:text-sm "
+            data-aos="fade-in"
+            data-aos-delay="1200"
+            data-aos-duration="2900"
+          >
+            sign up
```

Comment:
This sign up button doesn't work when signed in, it just sends you back to the homepage. Maybe change it to go to account when signed in?